Makefile

```diff
@@ -52,3 +60,11 @@ test-nll-%: TESTS=../tests-nll
 test-nll-%: OPTIONS += --nll
 test-nll-%:
 	cd charon && cargo run $(SRC)/$*.rs $(OPTIONS)
+
+ocaml-tests:
```

Comment:
also phony and the one below too: make is not aware of what it takes to regenerate dune builds